go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/Azure/go-ntlmssp v0.0.0-20191115210519-2b2be6cc8ed4 // indirect
 	github.com/ChrisTrenkamp/goxpath v0.0.0-20190607011252-c5096ec8773d // indirect
 	github.com/IBM-Cloud/bluemix-go v0.0.0-20210611051827-cdc80c935c05
+	github.com/IBM-Cloud/power-go-client v1.0.56
 	github.com/IBM-Cloud/terraform-provider-ibm v1.26.2
```

Comment:
i see packages changes in go.mod but no vendored files?

Reply:
ah, right. i wasn't sure what the process was for that with this repo. do you know what the process is? just run `go mod vendor` and check in changes?

Reply:
yes. `go mod vendor` should be fine.